data-prepper-plugins/aws-lambda/src/main/java/org/opensearch/dataprepper/plugins/lambda/common/config/BatchOptions.java

```diff
@@ -7,21 +7,24 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
 
 
 public class BatchOptions {
+    static final String DEFAULT_KEY_NAME = "events";
 
-    private static final String DEFAULT_BATCH_KEY = "events";
-
-    @JsonProperty("batch_key")
-    private String batchKey = DEFAULT_BATCH_KEY;
+    @JsonProperty("key_name")
+    @Size(min = 1, max = 2048)
```

Comment:
Is max value on the `key_name` consistent across data prepper for other process ?

Reply:
yes